### DISCUSSION TITLE
Sorting option suggestions.

### DISCUSSION THREAD

**Discussion by TheT113** [TeamRizu/OutFox#249] [03 - Feature Requests]
[Copy and pasted from the Discord, so sorry if it's a little rambly]:

some of the neat sorting stuff [the theme] D3NEX_MAXIMUM does that I'd love to see: you can set what order the Groups appear in, so I could do say: All the DDR Arcade releases in order, then the home releases, then all the packs from one author/site in release order; so that way, it doesn't get jumbled up by weird alphabetical namings. (actually, I think D3 does this instead by, sorting them in alphabetical order, but then you can rename them whatever you want, so it doesn't have to match the folder name). You can also put in a text document in each group to sort all the songs in a specific order, and then color their names specific colors (as a way of grouping songs, or dividing up the Dancemania songs from the Konami originals, etc.) I'd love to see the latter as an option somehow.

EDIT: Here's actually a page with a bunch of, how these themes (waiei-based themes) do it: https://sm.waiei.net/other/group.html

**Comment by Scraticus** [comment]
could you take a moment to edit this so it's in the proper [feature request] format for me?
